mslib/msui/multilayers.py

```diff
@@ -0,0 +1,470 @@
+from PyQt5 import QtWidgets, QtCore
+import logging
+import mslib.msui.wms_control
+
+
+class Multilayers(QtCore.QObject):
+    needs_repopulate = QtCore.pyqtSignal()
+
+    def __init__(self, docker_widget):
+        super().__init__()
+        self.parent = docker_widget
+        self.layers = {}
+        self.layers_priority = []
+        self.current_layer: Layer = None
+        self.save_data = True
+        self.synced_reference = Layer(None, None, None, is_empty=True)
+        self.parent.listLayers.itemChanged.connect(self.multilayer_changed)
+        self.parent.listLayers.itemClicked.connect(self.multilayer_clicked)
+        self.parent.listLayers.itemDoubleClicked.connect(self.multilayer_doubleclicked)
+        self.parent.listLayers.setVisible(True)
+        self.parent.leMultiFilter.setVisible(True)
+        self.parent.lFilter.setVisible(True)
+        self.parent.leMultiFilter.textChanged.connect(self.filter_multilayers)
+        self.parent.listLayers.setColumnWidth(1, 50)
+        self.parent.listLayers.header().setSectionResizeMode(0, QtWidgets.QHeaderView.Stretch)
+        self.parent.listLayers.setContextMenuPolicy(QtCore.Qt.CustomContextMenu)
+        self.parent.listLayers.customContextMenuRequested.connect(self.right_clicked)
+
+    def get_current_wms(self):
+        if self.current_layer:
+            return self.layers[self.current_layer.wms_name]["wms"]
+        return None
```

Comment:
return None is default. If you don't add a return None also None is returned :)

Any method or function returns always None if there is nothing else defined.